### DISCUSSION TITLE
re-encoding recorded video files on different computer

### DISCUSSION THREAD

**Discussion by hellomynameisleo** [blakeblackshear/frigate#9069] [Ask A Question]
I use video/audio copy on the machine frigate is hosted on. I have access to the video recording files on a more modern powerful computer that has a 13900k cpu, would re-encoding the hundreds of video files in the frigate recordings directory cause any playback issues if I kept the same video file names?

**Comment by NickM-27** [comment]
It would not cause issues, but would be more efficient to encode using go2rtc on the other machine as a proxy and then have the frigate machine record from there.

**Comment by hellomynameisleo** [comment]
> It would not cause issues, but would be more efficient to encode using go2rtc on the other machine as a proxy and then have the frigate machine record from there.

I had some issues before with restreaming to the machine hosting to frigate using ffmpeg but have encoutered some issues with errors in ffmpeg. So would rather just re-encode those stored videos, as re-encoding the video files won't cause issues does frigate work with x265 10-bit video files which I plan to encode to?

**Reply by NickM-27** [reply]
Frigate doesn't care, it has to do with whether or not the browser supports it
